web/.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "airbnb",
   "rules": {
+    "arrow-parens": ["error", "as-needed"],
```

Comment:
I don't have strong feelings about this rule, except that I've gotten used to the Airbnb default.  Now it looks strange to me to see an arrow function with a blocked body where the argument isn't wrapped in parens.  😛

Reply:
Yeah, this is one the casualties of war when using `prettier` -- the paren isn't necessary for single args so it's removed... there aren't too many conflicts between airbnb and prettier but there are a couple, but hopefully none are egregious, and keep me honest if you strongly object